packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -63,6 +77,10 @@
       margin: 0;
       position: fixed;
       right: 15px;
+
+      :global(.site-alert-active) & {
+        bottom: calc(var(--terra-consumer-site-alert-height, 30px) + 15px);
```

Comment:
not super thrilled with this approach, but its the best solution I can think of right now